lib/shelley/test/integration/Main.hs

```diff
@@ -141,9 +146,11 @@ instance KnownCommand Shelley where
 main :: forall t n . (t ~ Shelley, n ~ 'Mainnet) => IO ()
 main = withUtf8Encoding $ withTracers $ \tracers -> do
     hSetBuffering stdout LineBuffering
+    ci <- isJust <$> lookupEnv "CI"
```

Comment:
If we were to check for a hydra-specific flag, buildkite would run in ≈35 minutes instead of 39. Not sure if we want to do that.